src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -419,11 +409,21 @@
                      :transform [{:translate-y content-translate-y}]}
                     {})}
            (case @active-tab
-             1 [scan-qr-code-tab qr-view-finder request-camera-permission]
```

Comment:
Removed `request-camera-permission` as it is already a separate function and we don't have to pass it as a prop.